src/ui/imageballoonpanelview.js

```diff
@@ -86,6 +83,16 @@ export default class ImageBalloonPanelView extends BalloonPanelView {
 	}
 
 	/**
+	 * @inheritDoc
+	 */
+	init() {
+		super.init();
```

Comment:
This may return a promise. Switch the order of calls in this function and return the result of `super.init()` – that'd be the easiest solution. And remember to write a test that a promise is returned ;)